### ISSUE TITLE
Documentation about `any` type does not reflect difference with `object`

### ISSUE THREAD

**Issue by SergeyTeplyakov** [Microsoft/TypeScript-Handbook#57]
Current documentation for `any` type does not reflect the difference between `any` and `Object`.

If you take a look [here](https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/Basic%20Types.md#any), you may find following example: 

```
var notSure: any = 4;
notSure = "maybe a string instead";
notSure = false; // okay, definitely a boolean
```

But this example shows only one part of the `any` nature: an ability to assign anything to the variable of type any. Looking into this example it is really hard to understand the difference between `any` and `Object`, because following code would behave in the same way:

```
var notSure: Object = 4;
notSure = "maybe a string instead";
notSure = false; // okay, definitely a boolean
```

Proper example should show that the TypeScript type checker gives up and allows to call any method or access any properties on the variable of type any without any errors:

```
var notSure: any = 4;
notSure.weCanCallAnyMethodWithoutAnError();
notSure = "maybe a string instead";
var f = notSure.andAccessSomeFields;
notSure = false; // okay, definitely a boolean
```

**Comment by mhegazy** [comment]
@SergeyTeplyakov, thanks for reporting this, a PR would definitely be appreciated.

**Comment by sandersn** [comment]
Fixed by #72.
